tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -79,6 +79,14 @@ Aws::Client::ClientConfiguration& GetDefaultClientConfig() {
         cfg.verifySSL = true;
       }
     }
+    const char* connect_timeout = getenv("S3_CONNECT_TIMEOUT");
```

Comment:
S3_CONNECT_TIMEOUT_MSEC?

Reply:
Yes - this makes more sense given the name of the underlying attribute being affected.